src/slider/Slider.ts

```diff
@@ -149,7 +111,7 @@ export default class Slider extends SliderBase<SliderProperties> {
 
 		return v('output', {
 			classes: [ this.theme(css.output), outputIsTooltip ? css.outputTooltip : null ],
-			for: `${this._inputId}`,
+			for: this._inputId,
```

Comment:
Is this string conversion not necessary?

Reply:
didn't seem so. let me double check

Reply:
`this._inputId` comes from `uuid()` thus is inferred as a string.